sky/skylet/job_lib.py

```diff
@@ -137,6 +145,11 @@ def add_job(job_name: str, username: str, run_timestamp: str,
 
 
 def _set_status_no_lock(job_id: int, status: JobStatus) -> None:
+    """Setting the status of the job in the database.
+
+    Call the `set_job_stated` at the first time setting the job status
+    to RUNNING so as to update the start_at field.
```

Comment:
I think we should add back the assert  as a safer programmatic check. See another comment for a proposal?